protocol/river-status-unstable-v1.xml

```diff
@@ -68,6 +68,14 @@
       <arg name="tags" type="uint" summary="32-bit bitfield"/>
     </event>
 
+    <event name="urgent_tags">
+      <description summary="urgent tags of the output">
+        Sent once binding the interface and again whenever the tag is set
+        as urgent.
```

Comment:
This description sounds like the event is send for tags individually.

What should happen when one tag is urgent and another tags also gets urgent? I believe the second `urgent_tags` event should contain both tags, but the description does not reflect this.

Also, is this supposed to be send again when a tag looses urgency (yes, because otherwise it isn't really useful)? Should also be in the description.

Reply:
So I should use an array like in `view_tags` instead right?

Reply:
Not what I had in mind initially, but I think it is a pretty good idea.